news/_posts/2021-11-15-form-theme-new-in-1.7.8.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title:  "Introducing a new Form Theme for 1.7.8"
+subtitle: "An easy way to render Symfony forms"
+date:   2021-11-12
+authors:  [ matks ]
+icon: icon-lab
+tags: 
+ - 1.7.8.x
+ - feature
+ - development
+---
+
+In PrestaShop 1.7.8, we have carried out a [refactoring project](https://github.com/PrestaShop/PrestaShop/issues/16482) that modifies how we render forms in Symfony pages thanks to a new Form Theme. In this article, we'll see the benefits of this system and how it can be used in modules.
```

Comment:
```suggestion
In PrestaShop 1.7.8, a [refactoring project](https://github.com/PrestaShop/PrestaShop/issues/16482) has been carried out that significantly simplifies how forms are rendered in Symfony pages. In this article, we'll cover the benefits of this new Form Theme, and how it can be used in modules.
```